Assets/LeapMotion/Scripts/HandPool.cs

```diff
@@ -14,29 +14,20 @@
 namespace Leap {
   public class HandPool :
     HandFactory
-
   {
-    public IHandModel LeftGraphicsModel;
-    public IHandModel RightGraphicsModel;
-    public IHandModel LeftPhysicsModel;
-    public IHandModel RightPhysicsModel;
+
+    [SerializeField]
+    public List<IHandModel> ModelCollection;
```

Comment:
We probably should make this field private (and keep the SerializeField attribute so it shows up in the inspector)